sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedProcessorItem.java

```diff
@@ -55,8 +58,14 @@ public JsonNode getPrevious() {
      * @return change feed metadata.
      */
     @Beta(value = Beta.SinceVersion.V4_37_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    @JsonIgnore
     public ChangeFeedMetaData getChangeFeedMetaData() {
-        return changeFeedMetaData;
+        synchronized (monitor) {
```

Comment:
We don't need this synchronization for the getter. We can just return a new instance. Worst case we create a couple of extra instances, but that's okay.

Reply:
Fixed in next iteration.